Library/Homebrew/requirements/x11_requirement.rb

```diff
@@ -12,13 +12,9 @@ class X11Requirement < Requirement
 
   def initialize(name = "x11", tags = [])
     @name = name
-    if /(\d\.)+\d/ =~ tags.first
-      @min_version = Version.create(tags.shift)
-      @min_version_string = " #{@min_version}"
-    else
-      @min_version = Version.create("0.0.0")
-      @min_version_string = ""
-    end
+    # no-op on version specified as a tag argument
+    tags.shift if /(\d\.)+\d/ =~ tags.first
+    @min_version = Version.create("2.1.0")
```

Comment:
@ilovezfs Thoughts on this minimum version: https://www.xquartz.org/releases/archive.html (given 10.5 and friends)

Reply:
2.7.11 if >= 10.6
2.6.3 if == 10.5
anything goes if < 10.5